### PR TITLE
feat: Adding new abi and updating bytecodes

### PR DESCRIPTION
**Pull request by rossneilson**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 

Closes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ ] My code follows the existing style of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any UI changes have been tested and made responsive for mobile views

### CODE REVIEW

File: src/Components/GuildSidebar/MemberInfoDropdown/MemberInfoDropdown.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const MemberInfoDropdown: React.FC<MemberInfoDropdownProps> = ({
   onShowStakeModal,
 }) => {
   const { t } = useTranslation();
-
+  console.log({ isRepGuild });
```
Comment (Kenny-Gin1) [inline src/Components/GuildSidebar/MemberInfoDropdown/MemberInfoDropdown.tsx:47]: 🧹 Remove `console.log`

**Review by Kenny-Gin1 — CHANGES_REQUESTED** [review]
Looks good! You have some failing tests which I believe comes from us removing the `setPermissions` in our `ERC20Guild` contracts. It might also be that it got renamed to `setETHPermissions` in other cases.

And also please make sure to write a short summary and tick the checkboxes whenever your PR is ready for review. Otherwise, mark it as a draft if it is not ready yet.

**Comment by Filipv95** [comment]
@rossneilson Is there anything specific I should check in this PR? 'How to test' would be helpful also

**Comment by rossneilson** [comment]
I think currently this is broken and I haven't managed to get back to it, let me mark as draft

**Pull request closed without merging**